content/blog/20241214_fluent-package-v5.2.0-has-been-released.md

```diff
@@ -0,0 +1,143 @@
+# fluent-package v5.2.0 has been released
+
+Hi users!
+
+We have released fluent-package [v5.2.0](https://github.com/fluent/fluent-package-builder/releases/tag/v5.2.0).
+fluent-package is a stable distribution package of Fluentd. (successor of td-agent v4)
+
+This release is a new release of v5.2 series.
+
+## Changes from fluent-package v5.1.0
+* Support upgrade fluentd service with zero-downtime
+* Update ruby to 3.2.6
+* Update bundled Fluentd to v1.18.0
+* Update bundled gems overall
+* Solved possibility of DoS through "NoMemoryError"
+* deb rpm: Fixed to not execute v4 restart migration process unexpectedly
+* msi: set `GEM_HOME`/`GEM_PATH` in fluentd.bat
+* Fixed SIGABORT error with fluent-plugin-systemd
+
+## Support upgrade fluentd service with zero-downtime
+Previously, when upgrading the fluent-package, you had to stop the fluentd service,
+install the new version of the fluent-package, then install any necessary plugins,
+and finally restart the fluentd service.
+
+Starting from fluent-package v5.2.0,
+we have introduced a new feature that automatically installs the required plugins during the upgrade process,
+and introduced the zero-downtime restart feature which allows you to upgrade fluentd without interrupting the logging service.
+
+The zero-downtime restart feature can be configured by `FLUENT_PACKAGE_SERVICE_RESTART` environemnt variable.
+Please refer to following section for more details.
+
+* Add `FLUENT_PACKAGE_SERVICE_RESTART` environmental variable
+  * This section explains `FLUENT_PACKAGE_SERVICE_RESTART` environment variable.
+* Automate Plugin Install for Update on Demand
+  * This section explains the mechanism that install the plug-ins automatically during the upgrade process.
```

Comment:
It would be better to align the wording of `plugin`/`plug-in`.

```suggestion
  * This section explains the mechanism that installs the plugins automatically during the upgrade process.
```